frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```diff
@@ -133,6 +132,7 @@
   "Add resource": "Add resource",
   "Error loading the tasks.": "Error loading the tasks.",
   "Unable to locate any tasks.": "Unable to locate any tasks.",
+  "This Pipeline cannot be visualized. Pipeline taskSpec is not supported.": "This Pipeline cannot be visualized. Pipeline taskSpec is not supported.",
```

Comment:
run `yarn i18n`

Reply:
This is being used in the PipelineBuilderVisualization

Reply:
Ah okay :+1:.